src/main/java/com/github/lunatrius/schematica/client/gui/control/GuiSchematicControl.java

```diff
@@ -241,7 +240,6 @@ public void drawScreen(final int mouseX, final int mouseY, final float partialTi
         drawCenteredString(this.fontRenderer, this.strMoveSchematic, this.centerX, this.centerY - 45, 0xFFFFFF);
         drawCenteredString(this.fontRenderer, this.strMaterials, 50, this.height - 85, 0xFFFFFF);
         drawCenteredString(this.fontRenderer, this.strPrinter, 50, this.height - 45, 0xFFFFFF);
-        drawCenteredString(this.fontRenderer, this.strLayers, this.width - 50, this.height - 165, 0xFFFFFF);
```

Comment:
This text was drawn behind the layers text field, so it couldn't be seen normally.  Plus, it's using the "mode" string rather than a label-like string.